packages/charts/src/chart_types/xy_chart/state/selectors/on_click_caller.ts

```diff
@@ -84,3 +94,32 @@ function tryFiringOnProjectionClick(
   onProjectionClick(values);
   return true;
 }
+
+function tryFiringOnAnnotationClick(
+  annotationState: AnnotationTooltipState,
+  onAnnotationClick: SettingsSpec['onAnnotationClick'],
+): boolean {
+  if (annotationState && onAnnotationClick) {
+    const rectangularAnnotations = [];
+    const lineAnnotations = [];
```

Comment:
nit

```suggestion
    const rects = [];
    const lines = [];
```

Making this the eventual object keys simplifies line 121. Unless there is a good case for this I tend to match the keys. Since the context of this function is already annotation (i.e. `tryFiringOnAnnotationClick`) I think this is just easier.

Reply:
7d7700d for changes thanks!